docs/templates/template_reference.md

```diff
@@ -815,8 +815,24 @@ The selected value can be changed in the profile (consult the actual variable fo
 
     -   **datatype** - data type of the sysctl value, eg. `int`.
 
-    -   **sysctlval** - value of the sysctl value, eg. `'1'`. If this
-        parameter is not specified, XCCDF Value is used instead.
+    -   **sysctlval** - value of the sysctl value. This can be either not
+        specified, or an atomic value, eg. `'1'`, or a list of values,
+        eg. `['1','2']`.
+        -   If this parameter is not specified, an XCCDF Value is used instead
```

Comment:
Maybe we should clarify that the  XCCDF Value used is based on the name of the sysctl option, i.e.: `sysctlvar`.
It was already not very clear before...